test/functional/Java11andUp/playlist.xml

```diff
@@ -24,31 +24,9 @@
 
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TestConfig/playlist.xsd">
 	<test>
-		<testCaseName>StringvalueOfCodePoint_JDK11_CompactStrings_Enabled</testCaseName>
+		<testCaseName>String_CompactStrings</testCaseName>
```

Comment:
This test name is no longer appropriate since the test includes both compact and non-compact. I know Lan didn't like JDK11 in the name, but perhaps String_Java11andUp is appropriate. @llxia

Reply:
I agree. IMO we should run the entire JCL_TEST_Java-Lang with compact strings enabled explicitly and disabled explicitly. Or at the very least run all the String, StringBuffer, StringBuilder, Integer, Long, and StringCoding tests in both variations.

Reply:
Makes sense, although if we run the entire JCL_TEST_Java-Lang a second time that may significantly increase the sanity build time.

Reply:
@Ali-Ni the summary of all this is that if you remove `<variation>-XX:-CompactStrings</variation>` this will be ready to merge.

Reply:
Done

Reply:
> IMO we should run the entire JCL_TEST_Java-Lang with compact strings enabled explicitly and disabled explicitly

Created #2029